src/wristpy/io/readers/readers.py

```diff
@@ -81,7 +81,7 @@ def geneActiv_loader(
 ) -> WatchData:
     """Load input data from GeneActiv .bin file using actfast.
 
-        This loads the acceleration, lux, battery voltage, and capsense data.
+        This loads the acceleration, lux, battery voltage, and temperature data.
```

Comment:
The return a few lines down still shows the old WatchData class.

Reply:
In the docstring for geneActiv_loader correct? Fixed if so.